src/UpdateTracker.js

```diff
@@ -57,11 +60,17 @@ function trackResource(url) {
   if (!webSocket) {
     const socketUrl = `${protocol.replace('http', 'ws')}//${host}/`;
     webSockets[host] = webSocket = new WebSocket(socketUrl);
+    const backOffDelay = delay || 1000;
```

Comment:
Should we just call the function parameter `backOffDelay` instead of `delay`.
We can also initialize it as `trackResource(url, retryAttempt = 0, backOffDelay = 1000)`.

Reply:
Done. I've moved the default initialization to the setBackoff function as trackResource is called in a number of places already.